app/models/like.rb

```diff
@@ -0,0 +1,8 @@
+class Like < ApplicationRecord
+  belongs_to :author, class_name: 'User', foreign_key: :user_id
+  belongs_to :post, class_name: 'Post', foreign_key: :post_id
+
+  def update_likes_counter
+    post.increment!(:likes_counter)
+  end
```

Comment:
**Hint** : Just like I mentioned in my second  comment, `after_save : update_likes_counter`